symfony/framework-bundle/5.1/src/Kernel.php

```diff
@@ -17,12 +17,20 @@ protected function configureContainer(ContainerConfigurator $container): void
         $container->import('../config/{packages}/'.$this->environment.'/*.yaml');
         $container->import('../config/{services}.yaml');
         $container->import('../config/{services}_'.$this->environment.'.yaml');
+
+        if (file_exists($path = dirname(__DIR__).'/config/services.php')) {
+            (require $path)($container->withPath($path), $this);
```

Comment:
IMHO the API will be better with something like
```
$container->loadFromPath($path, $this);
```

Reply:
I'm not sure this makes sense: there is already $container->import() for this purpose. We don't need and API, and we don't *want* to maintain any.
Passing the kernel as the second argument is totally adhoc wiring.